src/containers/Bounty/SubmissionsAndCommentsCard.module.scss

```diff
@@ -52,6 +52,15 @@
   margin: $xxl-spacing auto;
 }
 
+.borderStyle {
+  li {
+    &:after {
+      border-bottom: $base-border !important;
```

Comment:
why these importants? We really should never have to use that...

Reply:
I'm not really sure how this got in here, this was from an old pr that I removed the `!important`s from 🤔